bin/hdfcoinc/pycbc_page_foundmissed

```diff
@@ -145,48 +146,56 @@ fig = plot.figure()
+                       label='missed', zorder=zmissed)
+
+fvals = vals[args.axis_type][found]
+
+ifsort = numpy.argsort(ifar_found)
+if args.missed_on_top:
+    ifsort = ifsort[::-1]
+
+fvals = fvals[ifsort]
+fdval = fdvals[ifsort]
+ifsorted = ifar_found[ifsort]
+
 if not args.gradient_far:
-    color = numpy.zeros(len(found))
-    ten = numpy.where(ifar_found > 10)[0]
-    hundred = numpy.where(ifar_found > 100)[0]
-    thousand = numpy.where(ifar_found > 1000)[0]
+    color = numpy.ones(len(found))
+    ten = numpy.where(ifsorted > 10)[0]
+    hundred = numpy.where(ifsorted > 100)[0]
+    thousand = numpy.where(ifsorted > 1000)[0]
     color[hundred] = 0.5
-    color[thousand] = 1.0
-
-    mpoints = plot.scatter(vals[args.axis_type][missed], mdvals, s=16, linewidth=0.5,
-                           marker='x', color='black', label='missed', zorder=zmissed)
-    points = plot.scatter(vals[args.axis_type][found], fdvals, s=16,
-                           c=color, linewidth=0, vmin=0, vmax=1,
-                           marker='o', label='found', zorder=zfound)
-    caption = (fig_title + ": Black x's are missed injections. "
-              "Blue circles are found with IFAR < 100 years, green are < 1000 years, and "
-              "red are found with IFAR >=1000 years. ")
+    color[thousand] = 0
+
+    norm = matplotlib.colors.Normalize()
+    caption = (fig_title + ": Red x's are missed injections. "
+              "Blue circles are found with IFAR < 100 years, gray are < "
+              "1000 years, and yellow are found with IFAR >=1000 years. ")
 else:
-    fvals = vals[args.axis_type][found]
-    color = 1.0 / ifar_found
-
-    # sort: if missed on top put high-FAR on top, if found on top vice versa
-    csort = color.argsort()
-    if not args.missed_on_top:
-        csort = csort[::-1]
-    fvals = fvals[csort]
-    fdval = fdvals[csort]
-    color = color[csort]
+    color = 1.0 / ifsorted
     if len(color) < 2:
         color=None
 
-    mpoints = plot.scatter(vals[args.axis_type][missed], mdvals, s=16, linewidth=0.5,
-                           marker='x', color='red', label='missed', zorder=zmissed)
-    points = plot.scatter(fvals, fdval, c=color, linewidth=0, s=16,
-                          norm=matplotlib.colors.LogNorm(),
-                          marker='o', label='found', zorder=zfound,cmap=args.colormap)
+    norm = matplotlib.colors.LogNorm()
     caption = (fig_title + ": Red x's are missed injections. "
-               "Circles are found injections. The color indicates the value of "
-               "the false alarm rate." )
-    plot.subplots_adjust(right=0.99)
+               "Circles are found injections. The color indicates the value "
+               "of the false alarm rate." )
+
+points = plot.scatter(fvals, fdval, c=color, linewidth=0, s=16, norm=norm,
+                      marker='o', label='found', zorder=zfound,
+                      cmap=args.colormap)
```

Comment:
Hi @GarethDaviesGW I don't see any arguments with colormap. Initially I recall I have put an option 

parser.add_argument('--colormap',default='cividis_r',
                   help="Type of colormap to be used for the plots.")

Such that anyone can switch the colourmap as they please. I wanted non-colourblind people to enjoy the spectrum of colours.

Reply:
Do you know if cividis without the _r reversal is okay? - I think that will look a bit better, but want to keep it suitable for as many as possible

Reply:
The markers were difficult to decipher without reversal.

Reply:
If I've understood the comment correctly, the colormap argument is still there - the file will be collapsed on the github display so that it doesn't have too much of the file which is unchanged by this PR

Reply:
One more thing args.colormap need parser.add_argument('--colormap') I don't see it here. Did I miss something?

Reply:
Ouch. I missed it. Approved

Reply:
It's still there on line 25: https://github.com/gwastro/pycbc/pull/3218/files#diff-c831f075d5a9aa6f90d34b98f9aef674R25